scripts/src/chartrepomanager/chartrepomanager.py

```diff
@@ -22,6 +22,7 @@
 sys.path.append('../')
 from report import report_info
 from chartrepomanager import indexannotations
+from signedchart import signedchart
 
 def get_modified_charts(api_url):
```

Comment:
Once pr is merged do we need to remove try section of removing the file path with vendor name?

Reply:
All code is required, nothing to remove after merge.